tests/framework/PostProcessors/Metrics/test_stats_metric.xml

```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0" ?>
+<Simulation verbosity="all">
+  <RunInfo>
+    <WorkingDir>scipyMetrics</WorkingDir>
+    <Sequence>mcRun, PP1</Sequence>
+    <batchSize>1</batchSize>
+  </RunInfo>
+
+  <TestInfo>
+    <name>framework/PostProcessors/Metric/test_metric_scipy</name>
+    <author>wangc</author>
+    <created>2018-02-16</created>
+    <classesTested>PostProcessors.Metric</classesTested>
+    <description>
+      This test checks metrics from Scipy library
+    </description>
+  </TestInfo>
+
+  <Files>
+    <Input name="output_metric" type="">output_metric.xml</Input>
+  </Files>
+
+  <Models>
+    <ExternalModel ModuleToLoad="../../../AnalyticModels/atten_and_poly.py" name="poly" subType="">
+      <variables>x1,x2,ans,ans2</variables>
+    </ExternalModel>
+    <PostProcessor name="pp1" subType="Metric">
```

Comment:
I tried to run this test, and it failed with the following error message:

<img width="826" alt="image" src="https://user-images.githubusercontent.com/7321071/175658659-34204a29-a80f-4ad3-93c7-1fb157466a0b.png">

In addition, this test is not tested since it is not added to the tests. @FlanFlanagan Could you resolve that?